server/statshouse/add_metrics_batch.cpp

```diff
@@ -0,0 +1,61 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2021 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include "server/statshouse/add_metrics_batch.h"
+
+#include <utility>
+
+#include "common/tl/constants/statshouse.h"
+#include "common/tl/methods/string.h"
+#include "common/tl/parse.h"
+#include "common/tl/store.h"
+
+namespace statshouse {
+
+constexpr int BUFFER_LEN = 1000;
+
+static int buffer[BUFFER_LEN];
+
+void StatsHouseMetric::tl_store() const {
+  tl_store_int(fields_mask);
+  tl_store_string(name.data(), name.size());
+  tl_store_int(0); // tags size
+  if (fields_mask & vk::tl::statshouse::metric_fields_mask::counter) {
+    tl_store_double(counter);
```

Comment:
here you write "counter", whereas below you write "this->field"

Reply:
fixed